module/VuFind/src/VuFind/Content/AbstractCover.php

```diff
@@ -171,8 +171,9 @@ public function getMetadata(?string $key, string $size, array $ids)
     }
 
     /**
-     * Which location are mandatory for backlinks, available locations are: detail,
-     * results
+     * Which location are mandatory for backlinks, available locations are the same
+     * as used for cover size determination, see
+     * https://github.com/vufind-org/vufind/blob/dev/config/vufind/config.ini#L864-L873
```

Comment:
These line numbers are going to be unstable over time; I'd suggest just saying "see coversize setting in config.ini" instead.

Reply:
I changed it

Reply:
Thanks!